src/ACEtk/interpretation/EL03/Bremsstrahlung.hpp

```diff
@@ -13,27 +13,27 @@ class Bremsstrahlung {
 				 { return entry * mega(electronVolt); } );
   }
 
-  auto photonEnergyRatio() const {
+  auto photonElectronEnergyRatio() const {
```

Comment:
Would it make sense to have an energyGrid method that returns a zip of electronEnergies and photonEnergyRatio - probably not because they are different lengths... Maybe a range of ranges. The point is that you need both to fully specify the two dimensions of what is currently called xsInterpolation. Ultimately, it would be nice to conform to the convention using energyGrid and values.